src/sections/target/Expression/GtexVariability.js

```diff
@@ -29,6 +36,13 @@ class GtexVariability extends Component {
 
   render() {
     const { theme, data } = this.props;
+    margin['left'] = getTextWidth(
+      data
+        .map(d => d.tissueSiteDetailId)
+        .sort((a, b) => b.length - a.length)[0],
```

Comment:
Can you move the logic of getting the longest id to a function such as:

```js
function getLongestId(data) {
  let longestId = '';
  data.forEach((d) => {
    if (d.tissueSiteDetailId.length > longestId.length) {
      longestId = d.tissueSiteDetailId;
    }
  });
  return longestId;
}
```

and use that function here. That way it's more readable what we are trying to do here and more performant since it's just a linear iteration to find the longest id? Thanks!

Reply:
Great suggestion! I changed it in [9dff77f ](https://github.com/opentargets/platform-app/pull/483/commits/9dff77f414eff33ae5fcee1ade75a4496f8ab5f3).